<launch>
     
     <!-- robot_state_publisher_ext -->	
     <node pkg="robot_state_publisher_ext" type="robot_state_publisher_ext_node" name="robot_state_publisher_ext" output="screen" >
       <param name="robot_name" type="string" value="coman" />
       <param name="urdf_path" type="string" value="$(find coman_urdf)/urdf/coman.urdf" />
       <param name="srdf_path" type="string" value="$(find coman_srdf)/srdf/coman.srdf" />
     </node>
     

</launch>
